tests/scripts/push.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source ${SCRIPT_DIR}/.definitions.sh
+source ${SCRIPT_DIR}/.local.sh
+
+REMOTE_PROJECT_FOLDER="~/${PROJECT}"
+
+# Copy over the contents of the project folder
+${SCRIPT_DIR}/sync.sh \
+        "${PROJECT_DIR}/" \
```

Comment:
Can we limit the amount of files we copy over to the remote instance? We only need the files in `test/`.